versions.props

```diff
@@ -10,6 +10,7 @@ com.palantir.remoting-api:* = 1.2.1
 com.palantir.safe-logging:safe-logging = 0.1.3
 com.squareup.okhttp3:* = 3.8.1
 com.squareup.retrofit2:* = 2.3.0
+io.dropwizard.metrics:metrics-core = 3.2.3
```

Comment:
3.2.4 is out now, see https://github.com/dropwizard/metrics/releases

@ellisjoe No specific concern right now, more just that it's another transitive library dependency that we're defining and potential diamond dependency conflicts in the future for folks that might have the metrics version imposed by Dropwizard server. I realize for most non-DW servers we'll specify it at the server level and set it in the BOM so should be fine.

Reply:
cool so I think the only way to pull this out into a separate module is to have remoting expose some way of injecting something that looks like an interceptor. I know when I talked to @uschi2000 awhile back we wanted to avoid that (reduce api surface), but maybe now we would be happy defining our own interceptor interface? or just something that is able to get a read-only view of requests/responses?

Reply:
wouldn't you want to instrument all clients anyway? or just those in WC?

Reply:
Ideally instrument all clients if we're happy accepting the diamond dependency. I think they've been good about backcompat so not too worried.